source/declarations.tex

```diff
@@ -8286,10 +8011,9 @@
 
 \pnum
 \indextext{specification!linkage!implementation-defined}%
-Every implementation shall provide for linkage to functions written in
-the C programming language,
+Every implementation shall provide for linkage to the C programming language,
 \indextext{C!linkage to}%
-\tcode{"C"}, and linkage to \Cpp{} functions, \tcode{"C++"}.
+\tcode{"C"}, and \Cpp{}, \tcode{"C++"}.
```

Comment:
It's not clear what the point of `, "C", and C++, "C++"` is - the sentence isn't valid English.

Here's my attempt at a rewrite of what I think the sentence is trying to say:
"Every implementation shall provide for linkage to the C programming language via C language linkage, and to C++ via C++ language linkage. The C and C++ language linkages can be explicitly specified using a /linkage-specification/ where the /string-literal/ is "C" or "C++" respectively."

Reply:
Yes. Too much of a rewrite for now, though.

Reply:
Deferred, PR welcome.